packages/forms/resources/views/components/key-value.blade.php

```diff
@@ -13,43 +13,55 @@
 >
     <div
         x-data="keyValueFormComponent({
-            state: $wire.{{ $applyStateBindingModifiers('entangle(\'' . $getStatePath() . '\')') }},
```

Comment:
This file looks pretty bad, @JohnathonKoster. 😂

Reply:
@zepfietje  Figured it out - good thing we test things first! 

If something fails horribly with the Pint integration, it can end up doing really dumb things. The fault has been corrected, and doing some tests in a private repo mirroring your changes things are working as intended. As an additional fail save, if the Pint integration fails, it will trigger some internal logic that will simply return the original template to avoid anything horrible from happening.

I would recommend a new branch/PR to account for first attempt removing some namespaces in some files (inside `@php @endphp`) regions. Would you like me to create a PR later today? Either way, make sure to use at least v1.5.3 🤛

Reply:
Thanks for debugging and fixing, @JohnathonKoster!
I'll revert the code style commit, update the plugin and rerun the formatting on this PR.